recipes/inja/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8.0)
```

```suggestion
cmake_minimum_required(VERSION 3.1)
```

CXX_STANDARD is CMake 3.0